index.js

```diff
@@ -28,6 +28,15 @@ function Cryptr(secret, options) {
         if (options.saltLength) {
             saltLength = options.saltLength;
         }
+
+        if (options.fixedIv) {
+            fixedIv = options.fixedIv;
```

Comment:
I think you're going to need to set `ivLength` here too so that `tagPosition` on line 42 is correct.